pom.xml

```diff
@@ -10,7 +10,7 @@
 
     <groupId>com.github.pires</groupId>
     <artifactId>hazelcast-kubernetes-bootstrapper</artifactId>
-    <version>3.8.7-SNAPSHOT</version>
+    <version>3.9-SNAPSHOT</version>
```

Comment:
Next time you don't need to change this as maven-release-plugin takes care of it  on release ;)